pishutdown.py

```diff
@@ -23,14 +27,17 @@ def buttonStateChanged(pin):
 
     if not (GPIO.input(pin)):
         # button is down
-        buttonPressedTime = datetime.now()
+        if buttonPressedTime is None:
```

Comment:
on second thought, the scenario I described would probably work fine... but still, I don't see the benefit of adding this check. am I missing something?